lib/utils.js

```diff
@@ -103,6 +103,23 @@ function postfixWithEndpointName(value, msg, definition) {
     }
 }
 
+function getSimulatedBrightnessKeys(options, msg, definition) {
```

Comment:
I don't think this function is necessary. The current keys are ok as the global store works per endpoint, not per device (so there won't be conflicts)